<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HMIS 105 Report</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0" rowspan="5">
        <widget class="QFrame" name="sidebar">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>260</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>REGISTER NEW PATIENT</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>AGE CATEGORY</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="comboBoxCategory">
              <item>
               <property name="text">
                <string>0 - 28 days</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>29 days - 4 years</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5 - 9 years</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10 - 19 years</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>20 years and above</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>SEX</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="comboBoxSex">
              <item>
               <property name="text">
                <string>Female</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Male</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>NEW ATTENDANCE</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="comboBoxNewAttendance">
              <item>
               <property name="text">
                <string>YES</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>NO</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>MONTH OF REPORT</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QDateEdit" name="dateEdit">
              <property name="displayFormat">
               <string>MMMM yyyy</string>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
              <property name="timeSpec">
               <enum>Qt::TimeSpec::UTC</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="IPN">
              <property name="placeholderText">
               <string>(Optional)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>IP NUMBER</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="vLayout">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Selected diagnoses</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="topMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QCheckBox" name="noClearForm">
                <property name="text">
                 <string>Don't clear Form</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkKeepDiagnoses">
                <property name="text">
                 <string>Keep Diagnoses</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListWidget" name="listWidgetSelected">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>120</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Select one or more diagnoses. Double click to add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit">
              <property name="placeholderText">
               <string>Type to filter diagnoses</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listWidgetAllDiagnoses"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>78</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnSave">
               <property name="text">
                <string>SAVE</string>
               </property>
               <property name="autoDefault">
                <bool>true</bool>
               </property>
               <property name="default">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnReset">
               <property name="text">
                <string>RESET</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QTableWidget" name="tableDiagnoses"/>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>HMIS DIAGNOSES FOR SELECTED MONTH</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkHideEmpty">
           <property name="text">
            <string>HIDE EMPTY ROWS</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Attendances</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTableWidget" name="tableAttendances">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>829</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHMIS">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionView_Register"/>
    <addaction name="actionExpand"/>
    <addaction name="actionToggle_Style"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuBackup">
    <property name="title">
     <string>Backup</string>
    </property>
    <addaction name="actionRestore"/>
    <addaction name="actionBackup"/>
   </widget>
   <addaction name="menuHMIS"/>
   <addaction name="menuBackup"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExpand"/>
   <addaction name="actionView_Register"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionExpand">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/toggle-on.png</normaloff>:/icons/toggle-on.png</iconset>
   </property>
   <property name="text">
    <string>Toggle SideBar</string>
   </property>
   <property name="toolTip">
    <string>Show or hide the sidebar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionView_Register">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/open-book.png</normaloff>:/icons/open-book.png</iconset>
   </property>
   <property name="text">
    <string>View Register</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionToggle_Style">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/theme.png</normaloff>:/icons/theme.png</iconset>
   </property>
   <property name="text">
    <string>Switch Theme</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/close-window.png</normaloff>:/icons/close-window.png</iconset>
   </property>
   <property name="text">
    <string>Exit HMIS</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="text">
    <string>Restore data from backup</string>
   </property>
  </action>
  <action name="actionBackup">
   <property name="text">
    <string>Create a database backup</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>IPN</tabstop>
  <tabstop>comboBoxCategory</tabstop>
  <tabstop>comboBoxSex</tabstop>
  <tabstop>comboBoxNewAttendance</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>listWidgetSelected</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>listWidgetAllDiagnoses</tabstop>
  <tabstop>tableAttendances</tabstop>
  <tabstop>tableDiagnoses</tabstop>
 </tabstops>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
